config/metric_config_sample.yaml

```diff
@@ -144,3 +144,23 @@ output:
       # you can add below any option supported by "Elasticsearch for Apache Hadoop"
       extraOptions:
         es.index.auto.create: false
+  - outputType: Hudi
+    outputOptions:
```

Comment:
whats mandatory and whats optional ? maybe clear the explanation for the delete part ?